src/carbonio-files-ui-common/views/components/UpdateNodeNameModalContent.tsx

```diff
@@ -114,6 +117,7 @@ export const UpdateNodeNameModalContent = <T extends UpdateNameMutation>({
 					onKeyUp={keyUpHandler}
 					hasError={!!errorMsg}
 					description={errorMsg || undefined}
+					CustomIcon={(): JSX.Element => <Text>.{ext}</Text>}
```

Comment:
When creating a folder, ext is undefined, but the dot is always shown

Reply:
+1

Reply:
I suggest to pass down the input CustomIcon as prop instead of ext